UPGRADE.md

```diff
@@ -8,16 +8,28 @@ does not have any particular instructions.
 
 ## Upcoming release
 
-### http api
+### **List of breaking changes**
+
+#### **Naming of Serviceless has changed**
+
+Currently, serviceless pods' name is generated based on the `Pod` name. `Kuma-cp` takes the pod's name and removes 2 last elements after splitting by `-`. This behavior is correct when `Pod` is managed by `Deployment` or `CronJob` but not for other types. We've introduced the change that takes the name from the parent resource which breaks current behavior.
```

Comment:
```suggestion
Currently, the `kuma.io/service` value of the inbound of a `Dataplane` generated for a `Service`less `Pod` is based on the `Pod` name. The Kuma CP takes the pod's name and removes 2 last elements after splitting by `-`. This behavior is correct when the `Pod` is owned by a `Deployment` or `CronJob` but not for other owner kinds. Kuma will now use the name of the owner resource as the `kuma.io/service` value.
```

Reply:
we need to be deliberate with how we explain this. a service is not a `Service`.